lib/browser.js

```diff
@@ -117,7 +118,16 @@ class Browser {
   }
 
   async _launch (preview) {
+    const isPkg = typeof process.pkg !== 'undefined'
+    const chromiumExecutablePath = (isPkg
+      ? puppeteer.executablePath().replace(
+          /^.*?\/node_modules\/puppeteer\/\.local-chromium/,
+          ospath.join(ospath.dirname(process.execPath), 'chromium')
+        )
+      : puppeteer.executablePath()
+    )
```

Comment:
(suggestion): We should probably explain why we are doing that and/or link to the corresponding documentation?

Reply:
Sure. I will do so.